addons/goldpinger/3.10.2-1.0.1/install.sh

```diff
@@ -0,0 +1,62 @@
+
+function goldpinger() {
+    local src="$DIR/addons/goldpinger/3.10.2-1.0.1"
+    local dst="$DIR/kustomize/goldpinger"
+    
+    # Check if upgrading from old okgolove chart
+    if is_old_chart; then
```

Comment:
this file is generated. you have to include the original `addons/goldpinger/template/base/install.sh`